system/include/emscripten/emscripten.h

```diff
@@ -262,6 +262,7 @@ double emscripten_asm_const_double(const char *code, ...);
 #if __EMSCRIPTEN__
 void emscripten_sleep(unsigned int ms);
 void emscripten_sleep_with_yield(unsigned int ms);
+void emscripten_block_on(char *identifier);
```

Comment:
Yeah, that's right. Good point about the documentation, I'll get on writing some when I find the time.